docs/How-to/Manage-a-Terra-validator/Set-up-validator.md

```diff
@@ -2,18 +2,19 @@
 
 This is a detailed step-by-step guide for setting up a Terra validator. Please be aware that while it is easy to set up a rudimentary validating node, running a production-quality validator node with a robust architecture and security features requires an extensive setup.
 
-::: tip
-Block42 has put together an excellent [step-by-step guide](https://medium.com/block42-blockchain-company/how-to-setup-a-terra-luna-validator-node-860d8ea7aea2) for setting up a new validator.
+::: tip Additional resources
+To supplement this guide, consider using Block42's helpful [step-by-step guide](https://medium.com/block42-blockchain-company/how-to-setup-a-terra-luna-validator-node-860d8ea7aea2) for setting up a new validator.
```

Comment:
This guide isn't up to date, or live to my knowledge.

Remove: "consider using Block42's helpful"

Replace: "Block 42 provide's their perspective on validator setup here" ...

Add line after line 6: "The most up to date information can be found in the documents you are currently reading, or on the [Terra Validator Discord] (https://discord.com/invite/xfZK6RMFFx)"

Consider adding this video where appropriate: https://www.youtube.com/watch?v=2lKAvltKX6w

Reply:
reworking this section.